pkg/driver/camera/camera.go

```diff
@@ -1 +1,5 @@
 package camera
```

Comment:
It would be nice to have a description of label generation rule in the godoc package comment.
```suggestion
/*
Package camera provides video camera driver.

Device Label Generation Rules

On Linux, device label will be in the format of:

	/dev/video0;/dev/v4l/by-path/pci-0000:00:00.0-usb-0:0:0.0-video-index0

If /dev/v4l/by-path/* is not available (for example in a docker container without
binding /dev/v4l/by-path/), it will be:

	/dev/video0;/dev/video0
*/
package camera
```

Reply:
Done! Adjusted some grammar.